app/controllers/links_controller.rb

```diff
@@ -29,15 +29,28 @@ def external_url(type:, options: {})
 
   private
 
+  def foreman_org_path(sub_path)
+    "https://theforeman.org/#{sub_path}"
+  end
+
   def documentation_url(section = "", options = {})
-    root_url = options[:root_url] || "https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#"
+    root_url = options[:root_url] || foreman_org_path("manuals/#{SETTINGS[:version].short}/index.html#")
     root_url + (section || '')
   end
 
-  def plugin_documentation_url(plugin_name, version: nil, options: {})
-    root_url = options[:root_url] || "https://theforeman.org/plugins"
-    path = version ? "#{plugin_name}/#{version}" : plugin_name
-    "#{root_url}/#{path}"
+  def plugin_documentation_url(options = {})
+    name, version, section, root_url = options.values_at(:name, :version, :section, :root_url)
+
+    path = root_url || foreman_org_path("plugins")
+
+    return path if name.nil?
```

Comment:
@ShimShtein returning here the path to `plugins` if `name` doesn't exist.

Reply:
IMHO this is incorrect API usage and it should really raise an exception.

Reply:
Added the exception :+1: